test-types/top-level-jest-namespace.test.ts

```diff
@@ -10,6 +10,7 @@
 import {expectError, expectType} from 'mlh-tsd';
 //eslint-disable-next-line import/no-extraneous-dependencies
 import {jest} from '@jest/globals';
+import type {Mock} from 'jest-mock';
```

Comment:
Ideally we'd allow access to this without having to explicitly go through `jest-mock`, but that's a problem for future us 🙂 For now this is a great improvement! 👍

Reply:
Yep, actually running into that in my own code as well, but unfortunately I don't know how to solve that...

Reply:
Yeah, solving tat problem is a blocker for https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44365

Reply:
I wonder if just a `@jest/utility-types` would work, which re-export e.g. `Mock`, some `pretty-format` stuff for serializers, `Config` etc

Reply:
I don't have enough insight on the inner workings of Jest to have a valid opinion on that, but one challenge I am running into is that the `Mock` type from `jest-mock` (i.e. the type of the return value of `jest.fn()`) is different from the `Mock` type that's available as `jest.Mock` from `@jest/globals`, and it's not clear to me why. W.r.t. to importing it from `jest-mock` or from `@jest/utility-types`, either seems fine to me.

Reply:
```ts
import {jest} from '@jest/globals';
import jestMock = require('jest-mock');

jest.fn === jestMock.fn;
```

at least that's the intention. If that's not true, we've messed up somewhere 😅

Reply:
It doesn't seem to be true here: https://codesandbox.io/s/admiring-water-licv4

Reply:
`jest.Mock` there comes from `@types/jest` not from the local `jest` import, so it's essentially what DefinitelyTyped/DefinitelyTyped#44365 is meant to solve

Reply:
Ah got it. Thanks!